packages/valko-ui-docs/src/pages/docs/ui/NotificationPage.vue

```diff
@@ -336,6 +341,50 @@ const createNotification = (props: NotificationProps) => {
         </div>
       </example-section>
 
+      <example-section
+        title="Gravity"
+        justify="start"
+        wrap
+        gap
+      >
+        <div
+          v-for="gravity in gravityOptions"
+          :key="gravity.value"
+          class="flex flex-col"
+        >
+          <span>{{ gravity.label }}</span>
+          <vk-button
+            :gravity="gravity.value"
+            class="mt-4"
+            @click="createNotification({text: gravity.label, gravity: gravity.value as any})"
```

Comment:
add spaces in all objects
```suggestion
            @click="createNotification({ text: gravity.label, gravity: gravity.value as any })"
```